djstripe/enums.py

```diff
@@ -0,0 +1,97 @@
+from enum import Enum as _Enum
+from django.utils.decorators import classproperty
+from django.utils.translation import ugettext as _
+
+
+class Enum(_Enum):
+    @classproperty
+    def choices(cls):
+        return tuple((cls.keys.get(k, k), v.value) for k, v in cls.__members__.items())
+
+    @classproperty
+    def keys(cls):
```

Comment:
I'd expect "keys" to behave more like the keys method on a dictionary (returns all keys). This method should default to returning a list of the enum's keys. I'd then override this in the other methods with a super call to replace keys with bad names.

Thoughts?